Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN mvn clean install --batch-mode -Plight-test
 
 # The image is based on https://github.com/jenkinsci/docker/tree/java11
 # All documentation is applicable
-FROM jenkins/jenkins-experimental:2.138.1-jdk11
+FROM jenkins/jenkins:jdk11
```

Comment:
LGTM, consistent with https://github.com/jenkinsci/jep/pull/216.
Though I wonder if this shouldn't be rather `2.152-jdk11`. Because if not, then any new possible build could yield a different result and stability?

Reply:
This is a `jenkins/jenkins-experimental` image. I do not care much about stability in this case. The main purpose of this image is to provide us with a quicker feedback cycle for any issues, and I believe it can also help to verify latest weekly releases of the Docker image

Reply:
I _guess_ that's fine, though I wonder if we shouldn't then name it `Dockerfile.experimental` or so so possibly users do not assume the `Dockerfile` in the official Jenkins repo is a way to get a Jenkins build OOTB.

Reply:
Custom Dockerfile names are not supported by DockerHub, and I am not ready to move the experimental build to trusted CI due to the lack of transparency there

https://github.com/jenkinsci/jenkins/blob/master/Dockerfile#L1-L3 is what I have to document it